### PR TITLE
chore(website): Getting Started copy

### PR DESCRIPTION
**Pull request by NicholasLYang**

### CODE REVIEW

File: website/src/_includes/docs/getting-started.md [hunk 1]
```diff
@@ -1,43 +1,60 @@
...
+		"format": "rome format ."
+	}
+}
+```
+
+Then you can run:
+
+```bash
+npm run format
+yarn run format
```
Comment (ematipico) [inline website/src/_includes/docs/getting-started.md:43]: Yarn doesn't need `run`

File: website/src/_includes/docs/getting-started.md [hunk 2]
```diff
@@ -1,43 +1,60 @@
...
 
-| Package manager               | Instructions         |
-|-------------------------------|----------------------|
-| [npm](https://www.npmjs.com/) | `npm i rome@next`    |
-| [pnpm](https://pnpm.io/)      | `pnpm i rome@next`   |
-| [yarn](https://yarnpkg.com/)  | `yarn add rome@next` |
+| Package manager               | Instructions            |
+|-------------------------------|-------------------------|
+| [npm](https://www.npmjs.com/) | `npm i -D rome@next`    |
+| [pnpm](https://pnpm.io/)      | `pnpm i -D rome@next `  |
```
Comment (ematipico) [inline website/src/_includes/docs/getting-started.md:26]: There's an additional space

File: website/src/_includes/docs/getting-started.md [hunk 3]
```diff
@@ -1,43 +1,60 @@
...
+Rome currently doesn't support other editors than VS Code. [Let us know](https://github.com/rome/tools/discussions/categories/suggestions) if you would like support for another editor.
 
 ### Install Rome CLI
 
-Install `rome` using your preferred node package manager.
+Install `rome` using your preferred node package manager. We require a minimum Node version of v14.18.
 
-> RECOMMENDATION: we advise to **not** install the binary globally, instead install the binary
-> locally to your project or use `npx/pnpx/yarn dlx` in order to leverage the CLI.
+> RECOMMENDATION: we recommend **not** installing the binary globally, instead please install the binary
```
Comment (ematipico) [inline website/src/_includes/docs/getting-started.md:19]: Isn't it repetitive to use recommend twice? We remove one, I think

**Review by ematipico — APPROVED** [review]

File: website/src/_includes/docs/getting-started.md (hunk 3 above)
Comment (NicholasLYang) [reply website/src/_includes/docs/getting-started.md:19]: We could change it to `NOTE:`

**Pull request merged**